libevm/JitVM.cpp

```diff
@@ -154,13 +148,23 @@ int64_t call(
 		//       the output is ignored here.
 		h160 addr;
 		std::tie(addr, std::ignore) = env.create(value, gas, input, {});
-		auto gasLeft = static_cast<int64_t>(gas);
+		o_result->gas_left = static_cast<int64_t>(gas);
+		o_result->release = nullptr;
 		if (addr)
-			std::copy(addr.begin(), addr.end(), _outputData);
+		{
+			o_result->code = EVM_SUCCESS;
+			// Use reserved data to store the address.
+			std::copy(addr.begin(), addr.end(), o_result->reserved.data);
```

Comment:
There was an `assert` to check that `addr` fits into `_outputData`, which you deleted. Maybe add here `assert` or `static_assert` that `o_result->reserved.data` is at least 20 bytes